### PR TITLE
readme: add documentation about the API diff feature

### PR DESCRIPTION
**Pull request by paulRbr**
This readme update should have been part of #35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,15 +21,110 @@ Bump is a Continuous Documentation Platform: it lets you keep your API doc alway
...
+        with:
+          doc: <BUMP_DOC_ID>
+          token: ${{secrets.BUMP_TOKEN}}
+          file: doc/api-documentation.yml
+          command: diff
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+```
+
+_Important: make sure to change your main destination branch name (`main` in the example above) and replace `<BUMP_DOC_ID>` with your Bump documentation slug or id._
```
Comment (Polo2) [inline README.md:60]: Maybe we could also remember here that user should change file with his own path ?

File: README.md [hunk 2]
```diff
@@ -46,28 +141,28 @@ jobs:
...
 ```
 
-Important: [actions/checkout](https://github.com/actions/checkout) has to be called **before this action**.
-
+_Important: make sure to change your main destination branch name (`main` in the example above) and replace `<BUMP_DOC_ID>` with your Bump documentation slug or id._
 
 ## Inputs
 
-* `doc` (required): Documentation id or slug. Can be found in the documentation settings on https://bump.sh
+* `doc` (required): Documentation slug or id. Can be found in the documentation settings on https://bump.sh
```
Comment (Polo2) [inline README.md:148]: Maybe we should favor direct access to front office (I'm quite sure that's what we do in other place but I can't find where)
```suggestion
* `doc` (required): Documentation slug or id. Can be found in the documentation settings on https://bump.sh/docs
```

**Review by Polo2 — APPROVED** [review]
Final step before we can release new version of Github action 🚀 
I just left two _tiny_ comments but already LGTM

File: README.md (hunk 2 above)
Comment (paulRbr) [reply README.md:148]: clearly!

File: README.md (hunk 1 above)
Comment (paulRbr) [reply README.md]: :+1: done

**Pull request merged**